submit_and_compare/static/html/submit_and_compare_view.html

```diff
@@ -11,7 +11,7 @@ <h2 class="problem_header">{display_name}</h2>
 		<div class='explanation'>{explanation}</div>
 	</div>
 	<div class='hint' style='display:none'>{{hint}}</div>
```

Comment:
could we move the style to the CSS file like we do for the `.button_holder.nodisplay`?